workspaces/ui-v2/src/optic-components/pages/diffs/CapturePage.tsx

```diff
@@ -145,9 +145,8 @@ function DiffCaptureResults() {
 
   const [handled, total] = handledCount;
 
-  // const history = useHistory();
-
-  const handleChangeToEndpointPage = (pathId: string, method: string) => () => {
```

Comment:
I'm actually extremely confused why this was working before - this is a double fn that wasn't called but apparently it was...????